aiida/orm/implementation/querybuilder.py

```diff
@@ -105,16 +105,6 @@ def __init__(self, backend: 'Backend'):
         type_check(backend, backends.Backend)
         self._backend = backend
 
-    @abc.abstractmethod
```

Comment:
Why remove this? It is implemented in `aiida/orm/implementation/sqlalchemy/querybuilder/main.py`.

Reply:
I just felt it was extraneous; we don't use it in the front-end QueryBuilder, and what was actually implemented in main (now removed) was returning "raw" (run-processed) results which would not be useful for the front-end.

Reply:
Ok fair enough